### DISCUSSION TITLE
Run command does not generate .raw file

### DISCUSSION THREAD

**Discussion by junyunPE** [nunobrum/PyLTSpice#91] [General]
Hello,
I am trying run an LTpisce simulation by LTC.run()  and then extract waveform data from raw file. But it doesn't create a new raw file and so there is an error when I try to read raw file.
<img width="428" alt="picture" src="https://user-images.githubusercontent.com/101173566/233311553-00c7b3f2-6c9b-46f3-b020-1fa69f1c4570.png">

**Comment by nunobrum** [comment]
Hello,

from your description, I tend to believe that there is something wrong with the simulation and the raw file is not created.
Can you verify that the .raw file exists ? You can consult the log file it was created ? When the script detects a problem, the .log file is renamed as .fail in order to identify the problems quicker.

I hope this helps,
Nuno Brum
